app/uk/gov/hmrc/agentclientrelationships/connectors/EnrolmentStoreProxyConnector.scala

```diff
@@ -67,7 +67,8 @@ class EnrolmentStoreProxyConnector @Inject()(
         else {
```

Comment:
if `userIds.isEmpty ` then throwing `RelationshipNotFound ` doesn't make sense, as we have not queried for any relationships in the ES call, but anyway that will be a big change  - not a concern to this PR